ros2/plugin_aspects.bzl

```diff
@@ -38,15 +38,16 @@ Ros2PluginCollectorAspectInfo = provider(
     ],
 )
 
-_ROS2_COLLECTOR_ATTR_ASPECTS = ["data", "deps"]
+# From https://github.com/bazelbuild/rules_python/blob/0.33.2/python/config_settings/transition.bzl#L121-L140
+_py_transition_attrs = ["target", "tools"]
```

Comment:
Do I understand this correctly that this is only needed for older rules_python than 1.x?

Reply:
The tag in the comment was the latest one at the time of writing. Later rules_python restructured that file so I don't immediately find the line on the current latest tag

Reply:
TBH, I've never really given too many thoughts about backwards compatibility. The project, at the moment, follows live-at-head philosophy and implicitly that also means that users of that repo have to accept that.